.github/workflows/ismrmrd_conda.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  pull_request:
+    branches:    
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build-conda-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: ismrmrd-build
+          environment-file: conda/environment.yml
+          python-version: 3.9
+          auto-activate-base: false
+      - name: Build conda package
+        shell: bash -l {0}
+        if: ${{ github.event_name == 'pull_request' }}
+        working-directory: conda
+        run: |
+          ./package.sh
+      - name: Push conda package
+        shell: bash -l {0}
+        if: ${{ github.event_name == 'push' }}
+        env:
+          ANACONDA_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
+        working-directory: conda
+        run: |
+          ./publish_package.sh -u ismrmrd -t "$ANACONDA_TOKEN" -p `find conda/build_pkg -name ismrmrd*.tar.bz2`
+
+  build-macos-conda-packages:
```

Comment:
Did you consider using a ["matrix strategy"](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) instead?

Reply:
This should actually be feasible here.

The only reason I separated out the macOS build into its own job, and did not use the build matrix, was that when I did that originally, I was not able to figure out how to filter the script and execution portion of the build configuration, by platform.  So if I used the matrix build strategy in the conda-recipes package, it would also then failed, b/c gadgetron would fail to build on macOS.

But since we now have parity on both platforms' builds, we should be able to use the built in matrix strategy, and just have 1 section/job for building the conda packages now for both platforms in parallel.